### PR TITLE
add recipe for conda-content-trust

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by wolfv** [comment]
@clee @awwad please let me know if you want to be co-maintainers of conda-content-trust in conda-forge.

**Comment by adriendelsalle** [comment]
OK for me

**Comment by wolfv** [comment]
@conda-forge/conda anyone interested to review this package? It's an optional dependency of conda since 4.10

**Comment by clee** [comment]
Wrong clee :)

**Comment by wolfv** [comment]
sorry! 

pinging @chenghlee :)

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
LGTM. Will wait a bit to see if Cheng or others have thoughts :)

File: recipes/conda-content-trust/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - pytest
+    - pytest-cov
+    - ruamel_yaml
+  commands:
+    - pytest -v tests
+    - conda-content-trust --help
+
+about:
+  home: https://github.com/conda/conda-content-trust
+  summary: Signing and verification tools for Conda"
```
Comment (chrisburr) [inline recipes/conda-content-trust/meta.yaml:40]:
```suggestion
  summary: Signing and verification tools for Conda
```

**Comment by wolfv** [comment]
I've discussed in private with @awwad and @chenghlee and they are both happy to be additional feedstock maintainers.

File: recipes/conda-content-trust/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set version = "0.1.3" %}
+
+package:
+  name: conda-content-trust
+  version: {{ version }}
+
+source:
+  url: https://github.com/conda/conda-content-trust/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 50a2732dcf3612bdff2b36171e3ebc72a74bbc258543a5aa2618218406b30a0d
```
Comment (awwad) [inline recipes/conda-content-trust/meta.yaml:9]: hash is correct

File: recipes/conda-content-trust/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+about:
+  home: https://github.com/conda/conda-content-trust
+  summary: Signing and verification tools for Conda
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - wolfv
+    - adriendelsalle
```
Comment (awwad) [inline recipes/conda-content-trust/meta.yaml:47]: Is this where one would add chenghlee & awwad?

**Review by awwad — APPROVED** [review]
LGTM.  securesystemslib is an optional dependency.

File: recipes/conda-content-trust/meta.yaml (hunk 3 above)
Comment (wolfv) [reply recipes/conda-content-trust/meta.yaml:47]: Yep, I added you both

**Pull request merged**